app/src/main/res/layout/activity_sticker_purchase.xml

```diff
@@ -4,6 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="#7F95D1"
```

Comment:
@SergeiMikhailovskii this color is better to move to `values/colors.xml` as well.